app/parse.py

```diff
@@ -15,8 +22,122 @@ class Product:
     num_of_reviews: int
 
 
+def init_driver() -> webdriver.Chrome:
+    options = webdriver.ChromeOptions()
+    options.add_argument("--headless")
+    options.add_argument("--no-sandbox")
+    options.add_argument("--disable-dev-shm-usage")
+    driver = webdriver.Chrome(options=options)
+    return driver
+
+
+def accept_cookies(driver: webdriver.Chrome) -> None:
+    try:
+        cookies_button = WebDriverWait(driver, 10).until(
+            ec.element_to_be_clickable(
+                (By.CLASS_NAME, "accept-cookies-button")
+            )
+        )
+        cookies_button.click()
+    except Exception:
+        pass
+
+
+def parse_products(driver: webdriver.Chrome) -> List[Product]:
+    products = []
+    product_elements = driver.find_elements(By.CLASS_NAME, "thumbnail")
+    for product_element in product_elements:
+        title = product_element.find_element(
+            By.CLASS_NAME, "title"
+        ).get_attribute("title").strip()
+
+        description = product_element.find_element(
+            By.CLASS_NAME, "description"
+        ).text.strip()
+
+        price = float(product_element.find_element(
+            By.CLASS_NAME, "price"
+        ).text.replace("$", "").strip())
+
+        rating = len(product_element.find_element(
+            By.CLASS_NAME, "ratings"
+        ).find_elements(By.CLASS_NAME, "ws-icon-star"))
+
+        num_of_reviews = int(product_element.find_element(
+            By.CLASS_NAME, "review-count"
+        ).text.split()[0])
+
+        products.append(
+            Product(
+                title=title,
+                description=description,
+                price=price,
+                rating=rating,
+                num_of_reviews=num_of_reviews
+            )
+        )
+
+    return products
+
+
+def load_more_pages(driver: webdriver.Chrome) -> None:
+    while True:
+        try:
+            more_button = WebDriverWait(driver, 10).until(
+                ec.element_to_be_clickable((By.CLASS_NAME, "btn-primary"))
+            )
+            more_button.click()
+            time.sleep(2)
+        except Exception:
+            break
```

Comment:
Similar to the previous comment, catching a general `Exception` without logging can obscure the reason why the loop is breaking. Consider logging the exception to provide insight into any issues encountered when clicking the 'Load More' button.